projects/fusion-ui/components/chip-filter/v3/chip-filter.component.scss

```diff
@@ -0,0 +1,125 @@
+@import '../../../src/style/scss/v3/mixins/mixins';
+@import '../../../src/style/scss/v3/vars/colors';
+@import '../../../src/style/scss/v3/vars/fonts';
+
+
+%chip-font {
+    letter-spacing: 0.24px;
+    font-size: 13px;
+    line-height: 15.87px;
+    font-family: $helveticaNueMd;
+    font-weight: 500;
+}
+
+$chipHeight: 32px;
+$chipMaxWidth: 352px;
+
+$paddingChip:  4px 12px 4px 12px;
+$paddingChipWithIcon:  4px 16px 4px 12px;
+$paddingChipWithCloseButton: 4px 10px 4px 14px;
+
+:host {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding:$paddingChip;
+    max-width: $chipMaxWidth;
+    width: fit-content;
+    height: $chipHeight;
+    @extend %chip-font;
+    border-radius: 100px;
+    border: none;
+    gap: 4px;
+    background-color: $fu-primary-100;
+    color:$fu-primary-500;
+
+    @include icon-style('fu-icon-close', $fu-primary-300, 16px, true, true,true){
```

Comment:
didn't we change the way we work with the icon component? why should still use ng-deep when we use icons?